src/app/static/src/tests/testHelpers.ts

```diff
@@ -112,14 +116,44 @@ export const expectChangeLanguageMutations = (commit: Mock) => {
     });
 };
 
-export const expectErrorReportOpen = (wrapper: Wrapper<any>, row = 0) => {
-    const link = wrapper.findAll(".dropdown-item").at(row);
-    link.trigger("click");
+export const expectErrorReportOpen = async (wrapper: any, row = 0) => {
+    const link = wrapper.findAll(".dropdown-item")[row];
+    await link.trigger("click");
 
-    expect(wrapper.find(ErrorReport).props("open")).toBe(true);
+    expect(wrapper.findComponent(ErrorReport).props("open")).toBe(true);
 }
 
 export function expectArraysEqual(result: any[], expected: any[]) {
     expect(result).toEqual(expect.arrayContaining(expected));
     expect(expected).toEqual(expect.arrayContaining(result));
 }
+
+// @ts-ignore
+export function shallowMountWithTranslate<T extends TranslatableState, C>(component: C, store: Store<T>, options?: any): VueWrapper<InstanceType<C>> {
```

Comment:
try extends partial defineComponent

Reply:
shallowMount unfortuantely complains about Partial<DefineComponent> and if I extend as DefineComponent the function is fine but then it complains about the components we put in are not the right type. May need to just add a comment about this!

Reply:
btw these aren't just components that use the `defineComponentVue2` it even complains about components that use `defineComponent`